oaknational/Oak-Web-Application
# feat(lesson overview page): create lesson overview page

## Description

- created a lesson overview page using fixture data

## Issue(s)

Fixes #1052 #1059

## How to test

1. Go to /beta/teachers/lessons/macbeth-lesson-1
3. You should see lesson  title; core content; empty expanding components for each lesson element; lesson specific brush border cards. 

## Screenshots


How it should now look:
![image](https://user-images.githubusercontent.com/48293828/207398660-d3925a22-e58e-4413-abc2-08b54cbc8186.png)
![image](https://user-images.githubusercontent.com/48293828/207398742-56a430bb-e0c9-45c7-8291-93d6a902691e.png)


## Checklist
